clang/include/clang/3C/3CGlobalOptions.h

```diff
@@ -0,0 +1,37 @@
+//=--3CGlobalOptions.h--------------------------------------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Tool options that are visible to all the components.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_3C_3CGLOBALOPTIONS_H
+#define LLVM_CLANG_3C_3CGLOBALOPTIONS_H
+
+#include "llvm/Support/CommandLine.h"
+
+extern bool Verbose;
+extern bool DumpIntermediate;
+extern bool HandleVARARGS;
+extern bool SeperateMultipleFuncDecls;
```

Comment:
There's a typo here: seperate->separate.  You might consider down the road alphabetizing these variables and adding comments.

Reply:
Filed correctcomputation/checkedc-clang#344.